src/general/OptionsMenu.cs

```diff
@@ -1276,7 +1281,9 @@ private void BuildInputRebindControls()
     private void OnOpenScreenshotFolder()
     {
         GUICommon.Instance.PlayButtonPressSound();
-        OS.ShellOpen(ProjectSettings.GlobalizePath(Constants.SCREENSHOT_FOLDER));
+
+        if (OS.ShellOpen(ProjectSettings.GlobalizePath(Constants.SCREENSHOT_FOLDER)).Equals(Error.FileNotFound))
```

Comment:
Doesn't normal equality comparison `==` work here?

Reply:
No worries. Changed to preferred style.